pkg/collect/http_test.go

```diff
@@ -217,7 +217,7 @@ func TestCollectHTTP_Collect(t *testing.T) {
 					CollectorName: "example-com",
 				},
 				Get: &troubleshootv1beta2.Get{
-					Timeout: 200 * time.Nanosecond,
+					Timeout: "200ms",
```

Comment:
This was `200 * time.Nanosecond` earlier, so that should be "200ns" I believe. The purpose is to have as a very quick timeout and have it fail on purpose.

Reply:
the server delay is 200ms so this would become flaky if exactly equal to delay.

Reply:
Good catch. I'll update the test case.